src/accounts/openid.js

```
@@ -0,0 +1,319 @@
+  } catch (err) {
+    console.error('OpenID grant failed:', err);
+    return { error: 'openid-grant-failed' };
+  }
+}
+
+export function getServerHostname() {
+  const auth = getAccountDb().first(
+    'select * from auth WHERE method = ? and active = 1',
+    ['openid'],
+  );
+  if (auth && auth.extra_data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)